tf-modules/cumulus-rds-tf/variables.tf

```diff
@@ -173,16 +173,88 @@ variable "rds_scaling_timeout_action" {
   default = "ForceApplyCapacityChange"
 }
 
+variable "enabled_cloudwatch_logs_exports" {
+  description = "Set of log types to export to CloudWatch Logs. For Amazon Aurora PostgreSQL, the only valid value is [\"postgresql\"]."
+  type = list(string)
+  default = []
+}
+
+variable "postgresql_log_retention_days" {
+  description = "Log retention period (days) for RDS PostgreSQL logs. Valid values: [0 1 3 5 7 14 30 60 90 120 150 180 365 400 545 731 1096 1827 2192 2557 2922 3288 3653]"
+  type        = number
+  default     = 30
+  validation {
+    condition = contains(
+      [
+        0, 1, 3, 5, 7,
+        14, 30, 60, 90,
+        120, 150, 180,
+        365, 400, 545, 731,
+        1096, 1827, 2192, 2557,
+        2922, 3288, 3653
+      ],
+      var.postgresql_log_retention_days
+    )
+
+    error_message = "Invalid log retention period. Must be one of: 0, 1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1096, 1827, 2192, 2557, 2922, 3288, 3653."
+  }
+}
+
+variable "db_log_min_duration_ms" {
+  description = "The threshold (in ms) for logging slow queries in RDS. Default to -1 (disabled)"
+  default     = -1
+}
+
 variable "db_parameters" {
   type = list(object({
-    name = string,
-    value = string,
+    name         = string,
+    value        = string,
     apply_method = string
   }))
   default = [
     {
-      name  = "shared_preload_libraries"
-      value = "pg_stat_statements,auto_explain"
+      name         = "shared_preload_libraries"
+      value        = "pg_stat_statements,auto_explain"
+      apply_method = "pending-reboot"
+    },
+    {
+      name         = "auto_explain.log_format"
+      value        = "json"
+      apply_method = "pending-reboot"
+    },
+    {
+      name         = "auto_explain.log_analyze"
```

Comment:
Will this and others auto_explain.* parameters add lots of overhead even if we db_log_min_duration_ms to -1?

Reply:
I think there's a slight bit of overhead in that the plugin is loaded where it wasn't before regardless of the config, but that *shouldn't* be detrimental.  Might want to include it in the  CHANGELOG?

Reply:
@ygliuvt  Something like:

"`pg_stat_statements` and `auto_explain` extensions are now loaded at cluster startup for database clusters using cumulus-rds-tf"